tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/MissingOperationsTestCase.java

```diff
@@ -0,0 +1,61 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.meta.token.NameTokens;
+import org.jboss.hal.resources.Ids;
+import org.jboss.hal.testsuite.Console;
+import org.jboss.hal.testsuite.category.Domain;
+import org.jboss.hal.testsuite.fragment.finder.FinderFragment;
+import org.jboss.hal.testsuite.fragment.finder.FinderPath;
+import org.jboss.hal.testsuite.fragment.finder.ItemFragment;
+import org.jboss.hal.testsuite.util.ConfigUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+
+
+@RunWith(Arquillian.class)
+@Category(Domain.class)
+public class MissingOperationsTestCase {
+
+    private String[] menuItems = {"Remove", "Copy", "Start", "Stop", "Suspend", "Resume", "Restart", "Reload"};
+
+    @Drone
+    private WebDriver browser;
+
+    @Inject
+    private Console console;
+
+    /**
+     * Test if drop down menu at Runtime -> Server Groups -> Server contains all items.
+     * HAL-1582
+     */
+    @Test
+    public void testMenuItems() {
+        StringBuilder missingItems = new StringBuilder();
+        FinderFragment fragment;
+        if (ConfigUtils.isDomain()) {
+            fragment = console.finder(NameTokens.RUNTIME, new FinderPath()
+                    .append(Ids.DOMAIN_BROWSE_BY, "server-groups")
+                    .append("server-group","sg-main-server-group"));
+        } else {
+            fragment = console.finder(NameTokens.RUNTIME);
+        }
+        ItemFragment itemFragment = fragment.column(Ids.SERVER_GROUP).selectItem(Ids.SERVER_GROUP);
+        for (String menuItem: menuItems) {
+            if (!itemFragment.dropdown().containsItem(menuItem)) {
+                if (!missingItems.toString().isEmpty()) {
+                    missingItems.append(", ");
+                }
+                missingItems.append(menuItem);
+            }
+        }
+        Assert.assertTrue("Menu should contains values " + missingItems.toString() + ". See HAL-1582.", missingItems.toString().isEmpty());
```

Comment:
This is confusing a little bit, you expect the missing items to contain a value in the message, but you're asserting that it's empty